app/views/statistics/users.html.haml

```diff
@@ -311,19 +311,21 @@
       %th Nombre de comptes
     - oldslot = ''
     - remaining = @stats.slot_size
+    - slice_parity = "slice_even";
     - @stats.by_state.each do |slot|
       - if oldslot != '' and slot["slot"] != oldslot
         - c = "purged"
         - cnt = remaining
         - remaining = @stats.slot_size
-        %tr
+        %tr{:class => slice_parity}
```

Comment:
For your information, Rails has a [cycle helper](http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html#method-i-cycle)